proto2ros_tests/CMakeLists.txt

```diff
@@ -45,9 +45,7 @@ proto2ros_generate(
   CPP_OUT_VAR cpp_sources
   INCLUDE_OUT_VAR cpp_include_dir
   APPEND_PYTHONPATH "${PROTO_OUT_DIR}"
-)
-add_dependencies(
-  ${PROJECT_NAME}_messages_gen ${PROJECT_NAME}_proto_gen
```

Comment:
This adds the dependency to the custom target that is downstream of the actual `proto2ros generate` command, so it doesn't really work.